core/src/test/java/org/apache/iceberg/avro/TestAvroReadProjection.java

```diff
@@ -49,4 +56,27 @@
 
     return Iterables.getOnlyElement(records);
   }
+
+  @Test
+  public void testAvroArrayAsLogicalMap() throws IOException {
+    Schema writeSchema = new Schema(
+            Types.NestedField.optional(0, "map", Types.MapType.ofOptional(2, 3,
```

Comment:
Looks like indentation is off here. The project uses 2 spaces per indent, and 2 indents for continuation lines like this. Could you fix this method please?

Reply:
Oh right, just updated it.